ports/librabbitmq/portfile.cmake

```
@@ -3,8 +3,8 @@ include(vcpkg_common_functions)
 vcpkg_from_github(
   OUT_SOURCE_PATH SOURCE_PATH
   REPO alanxz/rabbitmq-c
-  REF ffe918a5fcef72038a88054dca3c56762b1953d4 # v0.10.0
-  SHA512 05756176feffc3ccff9bf4f8416191c382c9e5b51d7dd72664cea0407c847d50cb4048b8669415d879396c0c7e8c1c38c65a66a98701e55de09afed893abc5a0
+  REF d416b8b16d196085106cfe137a0ff6919a9f6752 # v0.10.0
```

```suggestion
  REF d416b8b16d196085106cfe137a0ff6919a9f6752 
```
Since the commit doesn't belong to v0.10.0. The comment should be removed here.